### PR TITLE
Service Fabric REST API specification version 6.2 RTO

### PR DESCRIPTION
**Pull request by vipul-modi**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes. **(Not Applicable)**
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/servicefabric/data-plane/Microsoft.ServiceFabric/stable/6.2/examples/GetServiceTypeInfoByName-1.json
Comment (VipulM-MSFT) [reply specification/servicefabric/data-plane/Microsoft.ServiceFabric/stable/6.2/examples/GetServiceTypeInfoByName-1.json]: thanks let me look in to it.

**Comment by VipulM-MSFT** [comment]
@annatisch - the issue should be resolved now. This should have been caught with our internal CI/build process on the Service Fabric swagger specification repo. We will improve this. @salameer

**Comment by annatisch** [comment]
Thanks @VipulM-MSFT.
There are a number of errors in both the model and linter PR_ONLY=true CI jobs, for example there are many properties with this:
```
"Property named: 'PackageSharingPolicy', for definition: 'DeployServicePackageToNodeDescription' must follow camelCase style. Example: 'packageSharingPolicy'.",
```
And many of the models:
```
Found errors in validating the response with statusCode "200" for x-ms-example "Get Correlated events" in operation "GetCorrelatedEventList".:
```

Could you take a look?

**Comment by VipulM-MSFT** [comment]
@annatisch - this is not an Azure API specifications so the naming suppressions are already part of the README.MD and approved @salameer .

**Comment by VipulM-MSFT** [comment]
We also suppress the example validations due to issues in the validation tooling. Once they are fixed we can re-enable the checks and those failures should turn in to blocking failures. @salameer has the context on the issues.

**Comment by samedder** [comment]
@VipulM-MSFT I believe the supressions need to be in the README.md at the root of all the versions, the same one that contains the tags for generation. Take a look at [this pr](https://github.com/darshanhs90/azure-rest-api-specs/tree/c880fd5151500106311b814d4d3cdf89e808a8c2/specification/authorization/resource-manager) that added supressions.

**Review by samedder — APPROVED** [review]

**Comment by VipulM-MSFT** [comment]
#hold-off 

There are genuine failures in the example validation, we will fix and update the PR with those and ensure that the suppressions are added at the root. Let's wait on merging this PR for now, until I and @samedder sign off again.

**Comment by VipulM-MSFT** [comment]
@salameer and @annatisch this is ready to go. All issues should be fixed now. #signoff

**Comment by VipulM-MSFT** [comment]
Ping - @salameer and @annatisch

**Comment by annatisch** [comment]
@VipulM-MSFT 
I can't leave comments on the diff itself because it's too large for Github to load it.
Can you please remove the newline characters from the end of descriptions, for example:
```
    "BackupSuspensionInfo": {
      "type": "object",
      "description": "Describes the backup suspension details.\n",
      "properties": {
        "IsSuspended": {
          "type": "boolean",
          "description": "Indicates whether periodic backup is suspended at this level or not."
        },
        "SuspensionInheritedFrom": {
          "$ref": "#/definitions/BackupSuspensionScope",
          "description": "Specifies the scope at which the backup suspension was applied.\n"
        }
      }
    },
```
This creates strange formatting in the generated SDKs for example:
```
class BackupSuspensionInfo(Model):
    """Describes the backup suspension details.
    .
```

**Comment by VipulM-MSFT** [comment]
@annatisch - the SDK generators should trim before generating the descriptions. I have removed the newlines. If there are no other issues, please publish the specification.

**Review by annatisch — APPROVED** [review]

**Pull request merged**
